Library/Homebrew/os/mac.rb

```diff
@@ -41,7 +41,7 @@ def full_version=(version)
 
     def latest_sdk_version
       # TODO: bump version when new Xcode macOS SDK is released
-      Version.new "11.0"
+      Version.new "11"
```

Comment:
I think the SDK version logic needs to remain the same because the SDKs (bizarrely) refer to minor versions so will fail until a newer CLT/Xcode is released. See `Library/Homebrew/os/mac/pkgconfig/11.0` for even more sadness 😭 

I don't think they can be just moved because they reference `/Library/Developer/CommandLineTools/SDKs/MacOSX11.0.sdk`

CC @Bo98 for any thoughts on this